apps/prairielearn/elements/pl-number-input/pl-number-input.py

```diff
@@ -289,22 +296,13 @@ def render(element_html: str, data: pl.QuestionData) -> str:
             html_params["missing_input"] = True
             html_params["parse_error"] = None
         else:
-            raw_submitted_answer = data["raw_submitted_answers"].get(name, None)
             if raw_submitted_answer is not None:
                 html_params["raw_submitted_answer"] = pl.escape_unicode_string(
                     raw_submitted_answer
                 )
         # Add true answer to be able to display it in the submitted answer panel
-        ans_true = None
-        if pl.get_boolean_attrib(
-            element, "show-correct-answer", SHOW_CORRECT_ANSWER_DEFAULT
-        ):
-            ans_true = format_true_ans(element, data, name)
-        if ans_true is not None:
-            html_params["a_tru"] = ans_true
-
-        partial_score = data["partial_scores"].get(name, {"score": None})
-        score = partial_score.get("score", None)
+        if show_correct_answer:
+            html_params["a_tru"] = format_true_ans(element, data, name)
```

Comment:
Aside, explicitly not requesting a change in this PR: it'd be nice to take a pass through all the elements and give things more sensible names. `a_tru` and `a_sub` are almost incomprehensible if you're not already familiar with them. AFAICT there's no reason to not just use `correct_answer` and `submitted_answer`.

Reply:
Yeah, some of the key names for the mustache template are just terrible...

Reply:
I can open a new PR for this.